### PR TITLE
high contrast version of logo

### PR DESCRIPTION
**Pull request by nicklee**
High contrast version of the logo when in high contrast mode:

Screenshot 2022-11-02 at 11 54 54

### CODE REVIEW

**Review by chris-lawton — APPROVED** [review]

File: wagtailio/static/sass/components/_footer-menu.scss [hunk 1]
```diff
@@ -120,6 +120,21 @@
...
+
+        @include media-query(large) {
+            margin-top: auto;
+            width: 155px;
+            height: 65px;
+            display: block;
+        }
+
+        @media (forced-colors: active) {
+            fill: $color--white; // force white if in high contrast mode
```
Comment (thibaudcolas) [inline wagtailio/static/sass/components/_footer-menu.scss:134]:
```suggestion
            fill: CanvasText;
```

Just using white will work with a dark theme but not with a light theme. We need to use a system color instead.

File: wagtailio/static/sass/layout/_header.scss [hunk 2]
```diff
@@ -25,6 +25,11 @@
             width: 153px;
             height: 65px;
         }
+
+        @media (forced-colors: active) {
+            fill: $color--white; // force white if in high contrast mode
```
Comment (thibaudcolas) [inline wagtailio/static/sass/layout/_header.scss:30]:
```suggestion
            fill: CanvasText;
```

Same as above

**Review by thibaudcolas — CHANGES_REQUESTED** [review]
Not working with a light theme (white on white). Always go for system colors when making adjustments for forced-colors mode.

**Review by thibaudcolas — APPROVED** [review]
have made the switch to `CanvasText` and make sure all system colors are allowed by linting.

**Pull request merged**